gctscript/wrappers/gct/exchange/exchange.go

```diff
@@ -83,17 +83,12 @@ func (e Exchange) Pairs(exch string, enabledOnly bool, item asset.Item) (*curren
 
 // QueryOrder returns details of a valid exchange order
 func (e Exchange) QueryOrder(exch, orderID string, pair currency.Pair, assetType asset.Item) (*order.Detail, error) {
-	ex, err := e.GetExchange(exch)
-	if err != nil {
-		return nil, err
-	}
-
-	r, err := ex.GetOrderInfo(orderID, pair, assetType)
+	o, err := engine.Bot.OrderManager.GetOrderInfo(exch, orderID, pair, assetType)
```

Comment:
This could potentially cause a race condition when other subsystems are updating the fields. BUT there are instances where changing this functionality will require methods to update status specifically like for this function in ordermanager cancel method:
![image](https://user-images.githubusercontent.com/9391715/97247489-e5b30900-1853-11eb-8364-0845fb0077a8.png)